### PR TITLE
CUB DAO- BLUE token update

### PR DESCRIPTION
**Pull request by lioncubdao**

### CODE REVIEW

File: terra/asset.json [hunk 1]
```diff
@@ -1117,7 +1117,15 @@
     "type": "cw20",
     "icon": "https://raw.githubusercontent.com/lioncubdao/logo/main/Screenshot%202023-04-10%20115645.png"
   },
-  {
+    {
+    "id": "terra1gwrz9xzhqsygyr5asrgyq3pu0ewpn00mv2zenu86yvx2nlwpe8lqppv584",
```
Comment (yusufaine) [inline terra/asset.json:1121]: @lioncubdao Hey Lion DAO team! There's already an existing entry for the `BLUE` token with the same `id` (around line 2001), could you update the necessary fields for that entry instead?

**Review by yusufaine — CHANGES_REQUESTED** [review]

File: terra/asset.json (hunk 1 above)
Comment (lioncubdao) [reply terra/asset.json]: we have updated the existing entry and deleted the other one, thank you :)

**Review by lioncubdao — COMMENTED** [review]
We have updated the already existing token details for BLUE, thank you :)

File: terra/asset.json [hunk 2]
```diff
@@ -1999,10 +1999,12 @@
   },
   {
     "id": "terra1gwrz9xzhqsygyr5asrgyq3pu0ewpn00mv2zenu86yvx2nlwpe8lqppv584",
+    "entity": "BLUE CUB DAO",
```
Comment (yusufaine) [inline terra/asset.json:2002]:
```suggestion
    "entity": "Lion Cub DAO",
```

@lioncubdao thanks for getting back on this so quickly, I just have one last question to clarify before merging the PR -- the `entity` field is mainly to "tag" a token to a specific project, and in this case, I'm assuming that the `BLUE` token belongs to your team, as such, would you like to update the entity to `Lion Cub DAO`? By doing so, the token would also show the associated telegram and twitter links.

See [`terra/entity.json`](https://github.com/coinhall/yacar/blob/b6e67dc004539353ca7b8126e64e5110f9493250/terra/entity.json#L150-L154)

**Review by lioncubdao — COMMENTED** [review]
we updated the entity of the project

**Pull request merged**